src/node/mini_miner.cpp

```diff
@@ -294,7 +297,7 @@ void MiniMiner::BuildMockTemplate(std::optional<CFeeRate> target_feerate)
         }
         // Track the order in which transactions were selected.
         for (const auto& ancestor : ancestors) {
-            m_inclusion_order.emplace(Txid::FromUint256(ancestor->first), sequence_num);
+            m_linearize_result.inclusion_order.insert({Txid::FromUint256(ancestor->first), sequence_num});
```

Comment:
In [mini miner]: Linearize should also return packages fees and sizes:

I think `emplace` is preferred, as it avoids making a copy.

Reply:
Yes updated